### DISCUSSION TITLE
Navbar not updating to current page

### DISCUSSION THREAD

**Discussion by goofbrush** [retypeapp/retype#505] [Q&A] [answered]
The left navbar wont highlight the current page, and also navigating to another page reloads the site. Im using a summary.md file to order the nav and it works normally when ran on localhost but breaks when ran through github pages.
Any help appreciated : )

https://goofbrush.github.io/body-harvest/
https://github.com/goofbrush/body-harvest

**Comment by geoffreymcgill** [comment, marked as answer]
I think the issue might be do to the empty links in your `Summary.md` file. 

Could you revised the empty links to be headers? The following sample demonstrates:

```md
- [Introduction](index.md)
- [Acquiring the emulator](Emulator.md)
- [Plugin Configuration](Config.md)
## Graphics plugins
- - [angrylion RDP Plus r8](Graphics/angrylion_RDP.md)
- - [shunyan's SoftGraphic 1.5.0](Graphics/shunyan_SoftGraphic.md)
- - [GLideN64 (softrender fix)](Graphics/GLideN64.md)
## Audio plugins
- - [Azimer's HLE v0.70 WIP 10](Audio/Azimer_HLE.md)
- - [shunyuan's HLE audio](Audio/shunyan_HLE.md)
## Additional stuff
## Savestates
```

See PR:
https://github.com/goofbrush/body-harvest/pull/1

As well, you can remove the empty `README.md` file as you are using `index.md` instead.

Hope this helps.

**Reply by goofbrush** [reply]
I tried both, but no luck. 
I also removed the folder structure and deleted summary.md as to use the default structure instead, then deleted the retype branch to let it repopulate, all with no success.

even this https://github.com/goofbrush/test has the issue, so Im assuming its coming from the workflow action?

**Reply by geoffreymcgill** [reply]
okay, I think I found the issue, although I am surprised the config error is causing the menu refresh problem.

In your `retype.yml` project config, revise the `url` config:

```yml
# url: goofbrush.github.io/bh-emu-guide/
url: goofbrush.github.io/body-harvest
```

As mentioned above, I'm a bit surprised this error in the `url` config is causing the refresh issue, but I was able to reproduce the issue and it was fixed by making that revision. Confirmed several times. I never noticed this issue in the past when the `url` was incorrect, so I'm not entirely sure what has changed. We're going to have to investigate. 

Hope this helps.

**Reply by goofbrush** [reply]
Oh wow nice catch. I had renamed the repo at some point and didnt even think to revise the url. Thanks for the help : )

**Reply by geoffreymcgill** [reply]
I see you revised the `url`, although it was revised to `http://` and should be `https`.

https://github.com/goofbrush/body-harvest/blob/main/retype.yml#L3

You don't actually need the `https://` as that is the default protocol assumed by Retype when not specified. 

Revising to the following is likely the best:

```
# url: http://goofbrush.github.io/body-harvest
url: goofbrush.github.io/body-harvest
```

Hope this helps.
